src/frontend/src/service/cookie-service.ts

```diff
@@ -30,4 +30,9 @@ export function setCookie(inputToken: string, inputId: string) {
 export function removeCookie() {
   document.cookie = 'authToken=; max-age=0;';
   document.cookie = 'userId=; max-age=0;';
+  document.cookie = 'oeci_token=; max-age=0;';
+}
+
+export function hasOeciToken() {
+  return decodeCookie().oeci_token;
```

Comment:
`hasOeciToken` sounds like it returns a boolean. So IMO it should return a boolean or be renamed to `oeciToken`

Reply:
While I was refactoring I realized that in this form it would return either a oeci_token `string` or `undefined` which satisfies the `if` statement. 

I will switch it back to returning `true/false`